packages/expo-analytics-amplitude/src/ExpoAmplitude.ts

```diff
@@ -0,0 +1,2 @@
+import { NativeModulesProxy } from 'expo-core';
+export default NativeModulesProxy.ExpoAmplitude;
```

Comment:
Since we do only `if (!ExpoAmplitude.methodName) {` checks, I guess something like this would be better:
```suggestion
export default NativeModulesProxy.ExpoAmplitude || {};
```